### PR TITLE
Add Chartbeat analytics to Story Pages

### LINKED ISSUE #6689: Add Chartbeat analytics to Story pages
**Is your feature request related to a problem? Please describe.**
We need to enable chartbeat analytics for story pages to enable us see how users interact with the story page content in real-time.

**Describe the solution you'd like**
Add `ChartbeatAnalytics` to the `StoryPage` using the approach below:
```
import ChartbeatAnalytics from '#containers/ChartbeatAnalytics';
...
<ChartbeatAnalytics data={pageData} />
```
@sareh suggests this may be the only work required, we can test as per the testing notes below and look at the alternative below if that proves not to be the case.

**Describe alternatives you've considered**
Chartbeat has already been integrated in simorgh so most of the work that needs to be done here will just ensure that it's working on the story pages for both canonical and amp. See an [example pr](https://github.com/bbc/simorgh/pull/5540) of how it was added to the MAP and Radio pages.

To get Chartbeat working for story pages, we need to update the current `ChartbeatAnalytics` [utils](https://github.com/bbc/simorgh/blob/latest/src/app/containers/ChartbeatAnalytics/utils/index.js) to accept the following properties for story pages.

| Property (param) |  |
|--------|-----------------------|
| `uid` (g) | 50924 - Unchanged |
| `domain` (h) | Unchanged - should be the same as other page types in that service  |
| `useCanonical` | Unchanged |
| `sections` (g0) | CPS has sections and categories that need to be considered. E.g. "Mundo, Mundo - internacional, Mundo - STY, Mundo - internacional - STY, Mundo - feature-category" Add logic to `buildSections` in utils to fit this case |
| `bbc_hid` | Unchanged | Unchanged |
| `title` | Promo Headlines: Headline (from AReS). Refactor `getTitle` to use the Headline for STY pages |
| `window._cbq` | `lgdin anon` as value. Always anon until/unless we integrate with Sign-In  |
| `referrer` (v) | Unchanged |

**Testing notes**
- View the page source to see the chartbeat tags
- Use the live HUD to see that the links are working etc.
- Look at the network call for the old and current

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc
- [X] This feature is expected to need manual testing.

**Additional context**

_As a BBC editor/journalist
I want Chartbeat tags added to story pages
So that I can see page usage in real time._

**_Acceptance Criteria_**
_Given I am on a World Service STY page
When I look at the network requests in the console
Then I can see hits to ping.chartbeat.com
And those hits contain the correct information about the Story_

### PR DESCRIPTION
**Pull request by FK78**
Resolves #6689

**Overall change:**
_Add Chartbeat analytics to Story Pages._

**Code changes:**

- _Add `ChartbeatAnalytics` to the `StoryPage`._
- _Update `ChartbeatAnalytics` to accept `STY` as a valid page type._
- _Update `ChartbeatAnalytics` tests._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by rhenshaw56 — APPROVED** [review]

File: src/integration/pages/storyPage/mundo/__snapshots__/amp.test.js.snap [hunk 1]
```diff
@@ -87,20 +87,13 @@ Object {
...
-    "beacon": false,
-    "image": true,
-    "xhrpost": false,
-  },
-  "triggers": Object {
-    "trackPageview": Object {
-      "on": "visible",
-      "request": "pageview",
-    },
+  "vars": Object {
```
Comment (12) [inline src/integration/pages/storyPage/mundo/__snapshots__/amp.test.js.snap:110]: Should these snapshots be changing? Looks like we're replacing ATI with Chartbeat here, when they should both be present, surely?

Comment (FK78) [reply src/integration/pages/storyPage/mundo/__snapshots__/amp.test.js.snap:110]: Fixed [here](https://github.com/bbc/simorgh/pull/6747/commits/63028a6b1988d245bb0e707e14e51dfdd9ba2978?file-filters%5B%5D=.jsx)

File: src/app/pages/StoryPage/index.jsx [hunk 2]
```diff
@@ -233,6 +234,7 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
         aboutTags={aboutTags}
       />
       <ATIAnalytics data={pageData} />
+      <ChartbeatAnalytics data={pageData} />
```
Comment (12) [inline src/app/pages/StoryPage/index.jsx:237]: Can we follow what other pages have done (using `ArticlePage` as an example here). By using `jest.mock` here:

https://github.com/bbc/simorgh/blob/115ac892008d74e896135d02c8e2e038c678c7b0/src/app/pages/ArticlePage/index.test.jsx#L22-L25

we end up with this in the snapshot.

https://github.com/bbc/simorgh/blob/115ac892008d74e896135d02c8e2e038c678c7b0/src/app/pages/ArticlePage/__snapshots__/index.test.jsx.snap#L6-L8

I think it's important that we capturing some form of Chartbeat implementation in these snapshots, and will also ensure we are keeping consistent with our other page types.

**Review by 12 — CHANGES_REQUESTED** [review]

File: src/app/pages/StoryPage/index.jsx (hunk 2 above)
Comment (FK78) [reply src/app/pages/StoryPage/index.jsx:237]: Fixed [here](https://github.com/bbc/simorgh/pull/6747/commits/9de378b28674720e7dad5d237f4f90b1685ceece)

**Review by 12 — APPROVED** [review]
Thanks for making those changes. Have checked Chartbeat dashboard and appears to be reporting ok.

**Comment by paruchurisilpa** [comment]
Rhys and Fahad have checked the presence of chatrbeat which seems to be fine..Can't test anything manually, so moving to Ready to merge..

**Pull request merged**